### PR TITLE
Update test suite to check upgradability for all versions

### PR DESCRIPTION
**Pull request by RobAtticus**

### CODE REVIEW

File: scripts/test_updates_all.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+set -o pipefail
+
+V1_TAGS="0.1.0 0.2.0 0.3.0 0.4.0 0.4.1 0.4.2"
+V2_TAGS="0.5.0 0.6.0 0.6.1 0.7.0-pg9.6 0.7.1-pg9.6 0.8.0-pg9.6"
```
Comment (cevian) [inline scripts/test_updates_all.sh]: add 0.1.0 to V2? worked before right?

Comment (RobAtticus) [reply scripts/test_updates_all.sh]: I added it, though I'm not sure I think it's for the best. It mostly works because we hadn't blocked things yet.

**Comment by RobAtticus** [comment]
We could but I'm not sure if the original intention was to test constraints or to test converting a table with those constraints.  I think the latter is a bit more useful in testing, and isn't easily composeable without duplicating the creation of the table anyway.

**Comment by erimatnor** [comment]
My suggestion is not about improving or changing the test. It is about making the tests more composable and easier to maintain. Creating the initial schema via only CREATE vs CREATE+ALTER should be equivalent from the test's standpoint.

**Comment by RobAtticus** [comment]
@erimatnor While I agree it should be the same, I'm not sure it is? Maybe I am mistaken but I thought @cevian mentioned there being several different code paths for achieving the same thing with regards to constraints, so I was just suggesting that the test is possiblu set up in a way to explicitly test the CREATE way, rather than CREATE+ALTER. I could be mistaken, so yes we should wait for @cevian

**Comment by cevian** [comment]
Having said the above I am pretty agnostic on how these scripts include or don't include each other, which is why I am approving.

**Comment by RobAtticus** [comment]
Will save composability for after 0.9.0

**Pull request merged**